src/api/app/views/webui/package/_rpmlint_result.html.haml

```diff
@@ -8,7 +8,9 @@
       - repo_arch_hash.each do |repository, architectures|
         = select_tag("rpmlint_arch_select_#{index}_#{repository}", options_for_select(architectures.reverse),
                      class: "rpmlint_arch_select_#{index} custom-select", onchange: "updateRpmlintDisplay('#{index}')")
-%pre.rpmlint-result.text-pre-wrap{ id: "rpmlint_display_#{index}" }
+-# haml-lint:disable InlineStyles
+%pre.rpmlint-result.text-pre-wrap{ id: "rpmlint_display_#{index}", style: 'max-height: 35rem;' }
```

Comment:
This inline style isn't needed if we have a more precise CSS rule for `.rpmlint-result` depending on the view where it's used. In the request show redesign, we could instead rely on `#rpmlint-log .rpmlint-result`.

Reply:
I thought we would get rid of this partial soonish. But it is not the case.

I will adjust the CSS rules.

Reply:
SCSS adjusted. Please have a look.